### PR TITLE
Set installation name on account creation and pairing

### PR DESCRIPTION
**Pull request by rasom**
status: ready

needed for https://github.com/status-im/status-mobile/issues/15944

### CODE REVIEW

File: protocol/requests/create_account.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ type CreateAccount struct {
 	// BackupDisabledDataDir is the directory where backup is disabled
 	BackupDisabledDataDir string `json:"backupDisabledDataDir"`
 
+	DeviceName         string `json"deviceName"`
```
Comment (ibrkhalil) [inline protocol/requests/create_account.go]: Typo, Should be `json:"deviceName"`

**Comment by ibrkhalil** [comment]
Should the title be `Set device name on account creation and pairing`?

File: protocol/requests/create_account.go (hunk 1 above)
Comment (rasom) [reply protocol/requests/create_account.go]: You are right, though what is weird is that it works anyway.

Comment (rasom) [reply protocol/requests/create_account.go]: https://cs.opensource.google/go/go/+/refs/tags/go1.20.5:src/reflect/type.go;l=1221 ok, the colon is not really needed by design, no miracles, the value will be detected by opening double quote https://cs.opensource.google/go/go/+/refs/tags/go1.20.5:src/reflect/type.go;l=1221

**Comment by rasom** [comment]
@ibrkhalil it is kind of the same thing here, and both names are already used in the code. It is a device name, but used fro installation as just a `name` property... Well, doesn't really change anything

File: protocol/requests/create_account.go (hunk 1 above)
Comment (ibrkhalil) [reply protocol/requests/create_account.go]: I think it's better to be consistent.

Comment (rasom) [reply protocol/requests/create_account.go]: well yep it is already fixed some time ago, I just wanted to verify whether what I tested could actually work

**Review by borismelnik — APPROVED** [review]
LGTM :) Is it required some changes on desktop app?

**Review by ibrkhalil — APPROVED** [review]

**Comment by rasom** [comment]
@borismelnik , no it shouldn't

**Review by alaibe — APPROVED** [review]

**Pull request merged**